alacritty.yml

```diff
@@ -429,13 +429,13 @@
 
 # Shell
 #
-# You can set `shell.program` to the path of your favorite shell, e.g.
+# You can set `shell.program` to the path of your favorite shell, e.g.,
 # `/bin/fish`. Entries in `shell.args` are passed unmodified as arguments to the
 # shell.
 #
 # Default:
-#   - (macOS) /bin/bash --login
-#   - (Linux/BSD) user login shell
+#   - (Linux/BSD/macOS) use the `$SHELL` environment variable, or the user's
+#     login shell if `$SHELL` is unset
```

Comment:
Shouldn't be necessary to document `$SHELL` here. The previous Linux/BSD stuff was just fine tbh.

Reply:
Hm... in the code `SHELL` takes precedence over the user's login shell though. Do you want to change that behavior?

Reply:
No the current behavior is correct. I guess we could keep this but I'd try to at the very least keep it in a single line.

Reply:
Sure, I'll shorten it a bit.